tests/arrays/chunk.test.js

```diff
@@ -1,17 +1,13 @@
-/*
-_.chunk(['a', 'b', 'c', 'd'], 2);
-// => [['a', 'b'], ['c', 'd']]
- 
-_.chunk(['a', 'b', 'c', 'd'], 3);
-// => [['a', 'b', 'c'], ['d']]
-*/
-
 const chunk = require('../../src/arrays/chunk');
 
-test(`array ['a', 'b', 'c', 'd'] is chunked into [[1, 2], [3, 4], [5]]`, () => {
+test(`array ['a', 'b', 'c', 'd'] is chunked by 2`, () => {
     expect(chunk(['a', 'b', 'c', 'd'], 2)).toStrictEqual([['a', 'b'], ['c', 'd']]);
 });
 
+test(`array ['a', 'b', 'c', 'd'] is chunked by 3`, () => {
+    expect(chunk(['a', 'b', 'c', 'd'], 3)).toStrictEqual([['a', 'b', 'c'], ['d']]);
+});
+
```

Comment:
I like how you separated the tests into several blocks. I have some more suggestions how to improve the readability of the tests:
- blocks of tests can be united with `describe`
- It is better to start tests name with `should` word
- Ideally the final tests should be read as documentation

Example what I mean:

```
describe("chunk", () => {
  test("should return splited arrays with remaining items in a separate array", () => {
    expect(chunk(['a', 'b', 'c', 'd'], 3)).toStrictEqual([['a', 'b', 'c'], ['d']]);
  });

  test("should return empty array if size of chunk is 0", () => {
    expect(chunk([1, 2, 3, 4, 5], 0)).toStrictEqual([]);
  });
...
```